pycbc/live/snr_optimizer.py

```diff
@@ -40,19 +42,23 @@ def compute_network_snr_core(v, *argv, raise_err=False):
     Compute network SNR as a function over mchirp, eta, and two aligned
     spin components, stored in that order in the sequence v.
 
-    Parameters:
-    - v (sequence): A sequence containing the input values for mchirp,
-                    eta, and spin components.
-    - *argv: Additional arguments required for the snr calculation.
-    - raise_err (bool, optional): A flag indicating whether to raise an
-                                  error if an exception occurs during
-                                  the computation. Defaults to False.
-
-    Returns:
-    - network_snr (float): The computed network SNR (Signal-to-Noise
-                           Ratio) value.
-    - snr_series_dict (dict): A dictionary containing the snr timeseries
-                              from each ifo.
+    Parameters
+    ----------
+    v : list
+        A list containing the input values for mchirp, eta, and spin
+        components.
+    *argv
```

Comment:
argv has to be a dictionary, right?

Reply:
That looks to be a specifically ordered list of arguments for the function, which I am not a fan of, but changing it is outside the scope of this PR

Reply:
Agreed, I feel like it somewhat defeats the purpose of the *argv being flexible in the number of arguments that can be passed and then forcing a specific number of ordered arguments within it!